src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java

```diff
@@ -70,21 +70,19 @@ static boolean hasInventory(@Nullable Block block) {
 
         Material type = block.getType();
 
-        switch (type) {
-            case CHEST:
-            case TRAPPED_CHEST:
-            case FURNACE:
-            case DISPENSER:
-            case DROPPER:
-            case HOPPER:
-            case BREWING_STAND:
-            case BARREL:
-            case BLAST_FURNACE:
-            case SMOKER:
-                return true;
-            default:
-                return SlimefunTag.SHULKER_BOXES.isTagged(type);
-        }
+        return switch (type) {
+            case CHEST,
+                TRAPPED_CHEST,
+                FURNACE,
+                DISPENSER,
+                DROPPER,
+                HOPPER,
+                BREWING_STAND,
+                BARREL,
+                BLAST_FURNACE,
+                SMOKER -> true;
+            default -> SlimefunTag.SHULKER_BOXES.isTagged(type);
+        };
```

Comment:
Just for future references, can you please add a comment here saying "TODO: Add designated SlimefunTag".
Because this switch case is really not needed for something that can be a tag, especially since nested tags are a thing, so shulker boxes can be a part of that one.